src/btree/bt_curprev.c

```diff
@@ -156,27 +166,26 @@ __cursor_fix_append_prev(WT_CURSOR_BTREE *cbt, bool newpage)
 		 * into our caller.  Anyway, if this code breaks for any reason,
 		 * that's the way I'd go.
 		 *
-		 * If we're not pointing to a WT_INSERT entry, or we can't find
-		 * a WT_INSERT record that precedes our record name-space, check
-		 * if there are any records on the page.  If there aren't, then
-		 * we're in the magic zone, keep going until we get to a record
-		 * number of 1.
+		 * If we're not pointing to a WT_INSERT entry (we didn't find a
+		 * WT_INSERT record preceding our record name-space), check if
+		 * we've reached the beginning of this page, a possibility if a
+		 * page had a large number of items appended, and then split.
+		 * If not, check if there are any records on the page. If there
+		 * aren't, then we're in the magic zone, keep going until we get
+		 * to a record number matching the first record on the page.
 		 */
-		if (cbt->ins != NULL &&
-		    cbt->recno <= WT_INSERT_RECNO(cbt->ins))
-			WT_RET(__cursor_skip_prev(cbt));
 		if (cbt->ins == NULL &&
-		    (cbt->recno == 1 || __col_fix_last_recno(page) != 0))
+		    (cbt->recno == page->pg_fix_recno ||
```

Comment:
@michaelcahill 
> The change looks right to me. Why haven't test/format prev tests hit this?

Before we started splitting column-store pages, the first page (recno == 1), was the only place this could ever happen, so the simple test was sufficient.